src/app/pages/auth/components/consent-page/consent-page.component.html

```diff
@@ -65,7 +65,8 @@
       <!--  Thank you slide, when not eligible-->
 
       <ion-slide id="thankyou">
-        <div id="logo-container">
+        <div style="height: calc(100% - 88px);">
```

Comment:
would it be possible to find a solution for this without mentioning px or make more adaptable for dynamic heights? What happens if the device is too small?

Reply:
The button at the bottom of the page needs 88px (including padding and margin) the rest of the height goes to logo and text block. I know this is not a good approach but otherwise, I need to restructure every component and restyle them.
It doesn't have a problem with a small device, for example, iPhone 5/SE. But for smaller devices yes we have problems.